Tasks/NodeTool/nodetool.ts

```diff
@@ -158,4 +182,35 @@ async function acquireNode(version: string): Promise<string> {
     return await toolLib.cacheDir(toolRoot, 'node', version);
 }
 
+// For non LTS versions of Node, the files we need (for Windows) are sometimes located
+// in a different folder than they normally are for other versions.
+// Normally the format is similar to: https://nodejs.org/dist/v5.10.1/node-v5.10.1-win-x64.7z
+// In this case, there will be two files located at:
+//      /dist/v5.10.1/win-x64/node.exe
+//      /dist/v5.10.1/win-x64/node.lib
+// This method attempts to download and cache the resources from this alternative location.
+// Note also that the files are normally zipped but in this case they are just an exe
+// and lib file in a folder, not zipped.
+async function acquireNodeFromFallbackLocation(version: string): Promise<string> {
+    console.log(`Attempting fallback location download for Node version: ${version}`);
+
+    let exeUrl: string = `https://nodejs.org/dist/v${version}/win-${os.arch()}/node.exe`;
+    let libUrl: string = `https://nodejs.org/dist/v${version}/win-${os.arch()}/node.lib`;
+
+    let exeDownloadPath: string = await toolLib.downloadTool(exeUrl, "node.exe");
+    let libDownloadPath: string = await toolLib.downloadTool(libUrl, "node.lib");
+
+    // Create a folder where we can copy both files to. The caching mechanism uses a directory and 
+    // caches that. We can't cache straight from the temp folder because there could be other files 
+    // there that we don't want to cache.
+    let tempDownloadDirectory: string = path.join(taskLib.getVariable('Agent.TempDirectory'), 'z');
```

Comment:
I wish we could gen a folder name to avoid conflicts or file locking, etc. But I hate to pull in uuid package just to gen a guid... Still I wish we could gen a folder name.